Docs/LinuxOsxInstall.md

```diff
@@ -13,15 +13,24 @@
 - Make sure you have OpenGL availible
 
 ## Linux installation instructions
+### Arch 
+The AUR package [`factorio-yafc-git`](https://aur.archlinux.org/packages/factorio-yafc-git) points to the original upstream project by ShadowTheAge. To make this package suitable for YAFC-CE take the following steps:
+1. Clone the AUR package
+2. Open PKGBUILD
+3. Edit the `source` to use the following URL: https://github.com/have-fun-was-taken/yafc-ce
+In the same folder, run `makepkg -sri` to build and install YAFC. Run with `factorio-yafc`. 
```

Comment:
```suggestion
Clone the AUR package [`factorio-yafc-ce-git`](https://aur.archlinux.org/packages/factorio-yafc-ce-git).
From the cloned folder, run `makepkg -sri` to build and install YAFC.

It installs a desktop file so you should be able to launch for your desktop environment. Alternatively, run from the terminal with `factorio-yafc-ce`. 
```

Reply:
So I spelled out the steps a little bit, due to the modification of the pkgbuild required. Given that there is now a pkgbuild dedicated to yafc-ce, which does not require modification, Im inclined to instead just point it out. Anyone on Arch should know how to install AUR packages, and anyone needing the instructions should probably get them from the Arch Wiki rather than our install guide.

Reply:
true